### ISSUE TITLE
Disable dotted canvas for taking screenshots

### ISSUE THREAD

**Issue by toco-cam** [camunda/camunda-modeler#4573]
### Problem you would like to solve

Taking screenshots quickly without dotted background

### Proposed solution

Disable the dots. The snap function of an invisible grid is sufficient to position elements.

### Alternatives considered

- Using the export function to generate a picture - which is significantly more work
- Option in the menu to disable dots displaying the grid on the canvas

### Additional context

Related to https://github.com/camunda/web-modeler/issues/11206
Also raised in [this conversation](https://camunda.slack.com/archives/C0693F1NFK5/p1728463367024459)

**Comment by nikku** [comment]
I agree this can be a valuable feature, it is simple to build, too, as the [grid infrastructure already supports toggling](https://github.com/bpmn-io/diagram-js-grid).

Moving to `backlog` and marking this as a spring cleaning topic, maybe we can pick it up as an easy pick one day.

**Comment by toco-cam** [comment]
Disabling the grid here - could be an option too
![CleanShot 2024-10-01 at 12 54 31](https://github.com/user-attachments/assets/af80c071-51da-459c-b7fa-754d47e6538e)

**Comment by philippfromme** [comment]
I'm also in favor of adding support for switching it on and off.

**Comment by HanselIdes** [comment]
Adding more buttons would add yet more complexity to the user experience. A hotkey to toggle it on/off could solve the job without adding complexity. Another option is to introduce 'modeling settings', which is where custom lint rules would also live

**Comment by nikku** [comment]
Came up again in [this conversation](https://camunda.slack.com/archives/C0693F1NFK5/p1728463367024459).

**Comment by nikku** [comment]
I looked into the different options today to restore the "no grid" feature, in a way accessible in our different modelers. **TLDR:** Grid is layout configuration, similar to the properties panel toggle state, and should be accounted for / preserved across all open tabs.

----

As laid out in earlier conversations, there is different options, sketched below:

| Option | Works in Desktop | Works in Web | As seen in (other tool) |
|:---|:---|:---|:---|
| Make configurable via Flags | ✔️ | ❌ |  |
| Make configurable via Settings | ✔️ | ❌/❓ |  |
| Make toggleable via keyboard shortcut | ✔️ | ✔️ | Miro (`CTRL/CMD+G`), Excalidraw (`CTRL/CMD+'`) |

---

Above analysis shows that we should go with the option to toggle it via keyboard shortcut, as also mentioned earlier (https://github.com/camunda/camunda-modeler/issues/4573#issuecomment-2387848546).

Combined with "persisted in context", similar to the properties panel toggle state, we end up with a solution very similar to the one that @hkupitz sketched [here](https://github.com/camunda/camunda-modeler/issues/3995#issuecomment-1825788117), one that integrates with our existing UX:

![Image](https://github.com/user-attachments/assets/91346c79-d41b-4aad-8aa2-037aeedee19b)

**Comment by nikku** [comment]
Will potentially be addressed via https://github.com/camunda/camunda-modeler/pull/5522.
